### PR TITLE
Add pirobase CMS and imperia CMS detection

### PR DESCRIPTION
**Pull request by pirobase-luca**
pirobase CMS is a content management system for enterprises. Example sites:
* [https://www.pirobase-imperia.com/](https://www.pirobase-imperia.com/)
* [http://www.meinestadt.de/](http://www.meinestadt.de/)
* [https://www.stadtwerke-erfurt.de/](https://www.stadtwerke-erfurt.de/)

imperia CMS is a workflow-oriented content management system. Example sites:
* [http://www.kicker.de/](http://www.kicker.de/)
* [https://www.berlin.de/](https://www.berlin.de/)
* [https://www.strato.de](https://www.strato.de)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -3646,6 +3646,17 @@
 			"script": "^immutable\\.(?:min\\.)?js$",
 			"website": "http://facebook.github.io/immutable-js/"
 		},
+		"imperia CMS": {
+			"cats": "1",
+			"meta": {
+			  "X-Imperia-Live-Info": "[A-Za-z-_0-9/]+",
```
Comment (jvoisin) [inline src/apps.json]: I don't think that it's useful to add the `[A-Za-z-_0-9/]+`, collisions with `X-Imperia-Live-Info` are super unlikely.

File: src/apps.json [hunk 2]
```diff
@@ -3646,6 +3646,17 @@
...
+			  "GENERATOR": "IMPERIA|imperia|Imperia"
```
Comment (jvoisin) [inline src/apps.json]: Please use `^` at the begining of your regexp and `$` at the end to speed up a bit the detection :)

File: src/apps.json [hunk 3]
```diff
@@ -3646,6 +3646,17 @@
...
+			},
+			"icon": "imperiaCMS.svg",
+			"implies": "Perl",
+		  	"url": "imperia\/md\/",
```
Comment (jvoisin) [inline src/apps.json]: I don't think that it's useful to escape slashes, is it?

**Comment by gadcam** [comment]
FYI patterns are case insensitive (see https://github.com/AliasIO/Wappalyzer/wiki/Specification#patterns), so you can simplify a bit the generator pattern for example.

File: src/apps.json (hunk 1 above)
Comment (pirobase-luca) [inline src/apps.json]: So I should just leave it empty?

File: src/apps.json [hunk 4]
```diff
@@ -6165,6 +6176,13 @@
 			"script": "//assets\\.pinterest\\.com/js/pinit\\.js",
 			"website": "http://pinterest.com"
 		},
+	    "pirobase CMS": {
+			"cats": "1",
+			"html": "[^<]+=\"[^\"]*site\/[.A-Za-z/0-9_-]*\/resourceCached\/[.A-Za-z/0-9_-]*\"[^>]*|[^<]+cbi:\/\/\/cms\/[^>]+",
```
Comment (gadcam) [inline src/apps.json]: The same apply here : <code>[.A-Za-z/0-9_-]</code> can be written as <code>[a-z\d/._-]</code> and slashes do not need to be escaped see the specification page ;)

File: src/apps.json [hunk 5]
```diff
@@ -3646,6 +3646,17 @@
 			"script": "^immutable\\.(?:min\\.)?js$",
 			"website": "http://facebook.github.io/immutable-js/"
 		},
+		"imperia CMS": {
+			"cats": "1",
+			"meta": {
+			  "X-Imperia-Live-Info": "",
+			  "GENERATOR": "^IMPERIA"
```
Comment (jvoisin) [inline src/apps.json]: It's apparently possible to get the version of imperia via this header (see [here]( https://www.strato.de/ ) for example.
Do you mind adding this?

File: src/apps.json (hunk 4 above)
Comment (pirobase-luca) [inline src/apps.json]: Thanks for the hint. \d should work, but the validation tool says it is wrong:
`Expecting 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '[', got 'undefined'`

File: src/apps.json (hunk 5 above)
Comment (pirobase-luca) [inline src/apps.json]: Just added this with the latest commit.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**